components/templates/TerminalChallenge/Info.tsx

```diff
@@ -0,0 +1,21 @@
+import { CopyableCode } from 'components/ui/CopyableCode'
+
+export const Info = ({
+  genesis,
+  copyableCodeContent
+}: {
+  genesis: any,
+  copyableCodeContent: string
+}) => {
+  return (
+    <div className="flex">
+      <div className="px-1 py-6 sm:px-12 text-white">
+        <div
+          className="genesis"
+          dangerouslySetInnerHTML={{ __html: genesis.body.html }}
+        ></div>
+        <CopyableCode code={copyableCodeContent} language='shell' />
```

Comment:
`CopyableCode` should be optional. If no `copyableCodeContent` provided, the component should not be shown. Not all challenges might have this piece of copyable code. Ideally we can make this part of the lesson content source file and dynamically inject it, but that will have to come later.